core/src/epicli/data/common/ansible/playbooks/roles/repository/files/download-requirements/centos-7/requirements.txt

```diff
@@ -36,7 +36,7 @@ elasticsearch-oss-6.8.5
 elasticsearch-oss-7.3.2 # Open Distro for Elasticsearch
 erlang-21.3.8.7
 ethtool
-filebeat-6.8.5 # actually it's filebeat-oss
+filebeat-7.8.1 # actually it's filebeat-oss
```

Comment:
is this comment still valid? I do not understand it. Maybe add link to some place explaining it or delete it.

Reply:
Not sure what is the purpose of this comment. I can remove it.

Reply:
I understand the change but maybe in order to clarify the lines should we change the name filebeat-7.8.1 to filebeat-oss-7.8.1? The same as in elasticsearch case. It is important to now which version we use (opensource code). It's just my proposal

Reply:
All Beats are oss so I don't think it's needed.

Reply:
Nice to know! Thanks:)